### DISCUSSION TITLE
ERROR WHEN TRY WITH PLUGINS

### DISCUSSION THREAD

**Discussion by obedmacallums** [danny-avila/LibreChat#1648] [Troubleshooting] [answered]
I always got the next error: 

Something went wrong. Here's the specific error message we encountered: Encountered an error while attempting to respond: 404 Unrecognized request argument supplied: functions

I put the correct apikey for dalle plugin

**Comment by danny-avila** [comment, marked as answer]
A user had the same issue recently:  https://discord.com/channels/1086345563026489514/1086345563026489517/1199322217029308567


> apparently, `gpt-3.5-turbo-0613` does not work with plugins (anymore?), have to choose `gpt-3.5-turbo-1106`  (or `gpt-3.5-turbo`) for the agent

Both DALL-E and Plugins are functional:

![image](https://github.com/danny-avila/LibreChat/assets/110412045/969ba7bc-4ac1-421a-bc6e-e99b7d875ccd)

**Comment by obedmacallums** [comment]
Thanks! and now I can get response from dalle plugin, but exactly the same image is showed at the chat, for example:

![image](https://github.com/danny-avila/LibreChat/assets/56170471/0263e4b1-7bfa-47a9-9668-3fdc5d764265)

is it normal?

**Reply by danny-avila** [reply]
Not normal it may happen time to time based on the LLM. I will look into it

**Reply by obedmacallums** [reply]
Thank you, danny

**Reply by danny-avila** [reply]
You’re welcome!

**Reply by obedmacallums** [reply]
Just for record, I'm using ghcr.io/danny-avila/librechat-dev-api:latest image and deploy-compose.yml file, because I did the deploy at an AWS instance
